ff -ruN LVM2.2.02.84.orig/doc/example.conf.in LVM2.2.02.84/doc/example.conf.in
--- LVM2.2.02.84.orig/doc/example.conf.in	2011-02-21 21:04:50.060780366 -0500
+++ LVM2.2.02.84/doc/example.conf.in	2011-02-21 21:05:45.267281673 -0500
@@ -308,7 +308,7 @@
     # Type 3 uses built-in clustered locking.
     # Type 4 uses read-only locking which forbids any operations that might 
     # change metadata.
-    locking_type = 1
+    locking_type = 3
 
     # Set to 0 to fail when a lock request cannot be satisfied immediately.
     wait_for_locks = 1

